src/app/AppInitGate.tsx

```diff
@@ -18,35 +19,41 @@ interface Props {
   children: React.ReactNode
 }
 
-const I18nGate = ({ loading, children }: Props) => {
+const AppInitGate = ({ loading, children }: Props) => {
   const changelanguage = useChangeLanguage()
   const allowOtaTranslations = useSelector(allowOtaTranslationsSelector)
   const otaTranslationsAppVersion = useSelector(otaTranslationsAppVersionSelector)
   const language = useSelector(currentLanguageSelector)
   const bestLanguage = findBestAvailableLanguage(Object.keys(locales))?.languageTag
 
-  const i18nInitResult = useAsync(
+  const i18nInitializer = async () => {
+    await initI18n(
+      language || bestLanguage || DEFAULT_APP_LANGUAGE,
+      allowOtaTranslations,
+      otaTranslationsAppVersion
+    )
+    if (!language && bestLanguage) {
+      await changelanguage(bestLanguage)
+    }
+  }
+
+  const initResult = useAsync(
     async () => {
-      await initI18n(
-        language || bestLanguage || DEFAULT_APP_LANGUAGE,
-        allowOtaTranslations,
-        otaTranslationsAppVersion
-      )
-      if (!language && bestLanguage) {
-        await changelanguage(bestLanguage)
-      }
+      Logger.debug('AppInitGate', 'Starting init')
+      await Promise.all([i18nInitializer(), ValoraAnalytics.init()])
+      Logger.debug('AppInitGate', 'init completed')
     },
     [],
     {
       onError: (error) => {
-        Logger.error('i18n', 'Failed init i18n', error)
+        Logger.error('AppInitGate', 'Failed init', error)
```

Comment:
super nit: Might as well make a TAG variable since we have three logs for 'AppInitGate'